Assets/Scripts/Game/EnemySounds.cs

```diff
@@ -69,7 +69,7 @@ void Start()
 
             RaceForSounds = (Entity.Races)Random.Range(1, 6);
 
-            // Start attrack timer
+            // Start attract timer
```

Comment:
Is this some kind of new romance option? You have 10 seconds to attract your enemy... lol

(hope you find my poking fun to be funny and not obnoxious)

Reply:
Hey, that wasn't even me who put that comment in :) I just figured I might as well fix the spelling error.

Reply:
Well it is now a correctly spelled English word. I suspect it should be attack rather than attract though - that I what I was pointing fun at.

Reply:
Oh, okay. well my 'joke' was really lame then. lol

Reply:
Haha, no worries :)

Reply:
This looks like one of my little idiosyncrasies / typos.